2024-02-12 19:03:05,505 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Configure stats pid to 343542
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Loading settings from /u3/oqcardos/.config/wandb/settings
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Loading settings from /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/settings
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-12 19:03:05,506 INFO    MainThread:343542 [wandb_init.py:_log_setup():526] Logging user logs to /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/run-20240212_190305-g0qk40k2/logs/debug.log
2024-02-12 19:03:05,507 INFO    MainThread:343542 [wandb_init.py:_log_setup():527] Logging internal logs to /u3/oqcardos/motion_prediction/per-motion-prediction/wandb/run-20240212_190305-g0qk40k2/logs/debug-internal.log
2024-02-12 19:03:05,507 INFO    MainThread:343542 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f26f01c8e50>
2024-02-12 19:03:05,507 INFO    MainThread:343542 [wandb_init.py:init():566] calling init triggers
2024-02-12 19:03:05,507 INFO    MainThread:343542 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'optimizer': 'AdamW', 'lr': 0.005, 'dataset': 'single-supports', 'epochs': 100}
2024-02-12 19:03:05,508 INFO    MainThread:343542 [wandb_init.py:init():616] starting backend
2024-02-12 19:03:05,508 INFO    MainThread:343542 [wandb_init.py:init():620] setting up manager
2024-02-12 19:03:05,526 INFO    MainThread:343542 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-12 19:03:05,530 INFO    MainThread:343542 [wandb_init.py:init():628] backend started and connected
2024-02-12 19:03:05,536 INFO    MainThread:343542 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-02-12 19:03:05,536 INFO    MainThread:343542 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-12 19:03:05,537 INFO    MainThread:343542 [wandb_init.py:init():720] updated telemetry
2024-02-12 19:03:05,567 INFO    MainThread:343542 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-12 19:03:05,870 INFO    MainThread:343542 [wandb_run.py:_on_init():2262] communicating current version
2024-02-12 19:03:05,918 INFO    MainThread:343542 [wandb_run.py:_on_init():2271] got version response 
2024-02-12 19:03:05,918 INFO    MainThread:343542 [wandb_init.py:init():804] starting run threads in backend
2024-02-12 19:03:09,612 INFO    MainThread:343542 [wandb_run.py:_console_start():2241] atexit reg
2024-02-12 19:03:09,612 INFO    MainThread:343542 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-02-12 19:03:09,612 INFO    MainThread:343542 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-02-12 19:03:09,612 INFO    MainThread:343542 [wandb_run.py:_redirect():2186] Redirects installed.
2024-02-12 19:03:09,613 INFO    MainThread:343542 [wandb_init.py:init():847] run started, returning control to user process
2024-02-12 19:03:09,617 INFO    MainThread:343542 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-12 19:03:09,617 INFO    MainThread:343542 [wandb_init.py:_pause_backend():437] pausing backend
2024-02-12 19:03:12,635 INFO    MainThread:343542 [wandb_init.py:_resume_backend():442] resuming backend
